### PR TITLE
Use unread count for room highlight state

### PR DESCRIPTION
**Pull request by dbkr**
Requires https://github.com/matrix-org/matrix-react-sdk/pull/68

### CODE REVIEW

**Comment by Kegsay** [comment]
LGTM - blocked on other PR before merging.

**Comment by dbkr** [comment]
this has changed so quick ptal then I'll merge?

File: src/components/structures/RoomSubList.js [hunk 1]
```diff
@@ -255,7 +255,7 @@ var RoomSubList = React.createClass({
                     collapsed={ self.props.collapsed || false}
                     selected={ selected }
                     unread={ self.props.activityMap[room.roomId] === 1 }
-                    highlight={ self.props.activityMap[room.roomId] === 2 }
+                    highlight={ (room.unread_notifications && room.unread_notifications['notification_count'] > 0) || self.props.label === 'Invites' }
```
Comment (Kegsay) [inline src/components/structures/RoomSubList.js]: I thought we had getters for this now?

**Comment by Kegsay** [comment]
LGTM aside from comment

**Comment by dbkr** [comment]
*ahem* apply patch the right branch...

File: config.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
-    "default_hs_url": "https://matrix.org",
+    "default_hs_url": "http://localhost:8008",
```
Comment (Kegsay) [inline config.json]: Oopsie :D

**Comment by Kegsay** [comment]
LGTM aside from config thing

**Pull request merged**